### DISCUSSION TITLE
Vyper code identified as Python

### DISCUSSION THREAD

**Discussion by c-protocol-team** [github-linguist/linguist#6477] [Classification] [answered]
the language stats for https://github.com/c-protocol/crayon-governance show 100% Python instead of 100% Vyper. Clicking on the stats correctly returns 0 files written in Python.

**Comment by DecimalTurn** [comment, marked as answer]
Looking at the [.gitattributes file](https://github.com/c-protocol/crayon-governance/blob/main/.gitattributes), there is an override that forces the extension `.vy` to count as Python:
```
*.vy linguist-language=Python
```

If this line is removed, the repo will appear as 100% Vyper in the stats bar. Also note that when you click on the stats bar, you end up on the search page which doesn't take gitattributes into account.

**Comment by c-protocol-team** [comment]
Thank you very much, Martin. That solved it.
Best regards.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
